src/ansys/tools/path/path.py

```diff
@@ -1156,7 +1156,11 @@ def version_from_path(product: PRODUCT_TYPE, path: str) -> int:
 
     """
     if not isinstance(path, str):
-        raise ValueError(f"Provided path '{path}' is not a string.")
+        raise ValueError(
+            f"Provided path '{path}' is not valid string."
+            f"Check if the {product} is installed in default location."
+            f"If not use ``save-ansys-path`` to save the path."
```

Comment:
Does this mean if the product is not installed in the default location then use "save-ansys-path"?

Reply:
Yes